tests/Common/DevBranchesTest.php

```diff
@@ -11,32 +11,66 @@
 
 class DevBranchesTest extends StorageApiTestCase
 {
-    private static $cleanupAfterClassTokenId;
-
-    private static $teardownClient;
-
-    /**
-     * @dataProvider provideValidClients
-     */
-    public function testCreateBranch(Client $providedClient)
+    public function setUp()
     {
-        $providedToken = $providedClient->verifyToken();
-        $branches = new DevBranches($providedClient);
+        parent::setUp();
 
-        // cleanup
-        $branchesList = $branches->listBranches();
-        $branchName = __CLASS__ . '\\' . $this->getName() . '\\' . $providedToken['id'];
-        $branchesCreatedByThisTestMethod = array_filter(
-            $branchesList,
-            function ($branch) use ($branchName) {
-                return strpos($branch['name'], $branchName) === 0;
+        // branches cleanup
+        $branches = new DevBranches($this->_client);
+        foreach ($branches->listBranches() as $branch) {
```

Comment:
Takhle to nemůže být, nebude to fungovat v paralelních testech. Musíš mazat jen ty branche, které jsou vytvořené tím kterým testem IMHO

Reply:
Asi bude potřeba to vytahnout do nějakého traitu a udělat nějaké `cleanupBranchesByPrefix` nebo tak něco...

Reply:
Možná vlastně tenhle problém máme i v tom našem, @fejjo - že když by běžely paralelně dva testy v té naší suitě tak si přemažou navzájem branche,...

Reply:
No ja som niekde v prvom PR testu pisal, ze som neriesil ci to moze bezat paralelne pretoze, to sa bude musiet aj tak vratit potom ten test do tej povodnej triedy ComponentTest aby sa to testovalo vsetko jednou triedou klasicky alebo pre dev branch

Reply:
Já bych to teď taky neřešil.
Protože až budeš chtít tenhle test paralelizovat tak to budeš muset stejně nějak upravit.
Nebudeš tam chtít aby ti to pro každý test do dev branche nahrálo milion test bucketů/konfigurací.

Reply:
Přijde mi zbytečné si přidělávat práci, až to budem paralelizovat. Ale OK